pkgs/native_toolchain_c/lib/src/cbuilder/cbuilder.dart

```diff
@@ -57,18 +57,43 @@ class CBuilder implements Builder {
   @visibleForTesting
   final Uri? installName;
 
+  /// Definitions of preprocessor macros.
+  ///
+  /// When the value is `null`, the macro is defined without a value.
+  final Map<String, String?> defines;
+
+  /// Whether to define a macro for the current [BuildMode].
+  ///
+  /// The macro name is the uppercase name of the build mode and does not have a
```

Comment:
Note to self: I found this in some build systems to be the default, but I didn't add documentation. 🙄